src/main/java/hudson/os/windows/ManagedWindowsServiceLauncher.java

```diff
@@ -100,18 +100,6 @@
      */
     private ManagedWindowsServiceAccount account;
     
-    public static class LogOnOption {
```

Comment:
Formally it's a binary compatibility break. Not sure if it's important at all in such case, but probably it makes sense to just deprecate it since it's just a refactoring